src/views/Search.vue

```diff
@@ -7,16 +7,26 @@
     </ion-header>
     <ion-content>
       <ion-searchbar @ionFocus="selectSearchBarText($event)" v-model="queryString" :placeholder="$t('Search')" @keyup.enter="queryString = $event.target.value; searchProducts()"/>
+  
+  
+     <!-- No result -->
+     <div class="empty-state"  v-if="products.length===0" >
+        <p>{{ $t("No results found")}}</p>
```

Comment:
Only single div tag with class empty-state should be there having p and img tags